pkg/operator/deploy/helpers.go

```diff
@@ -0,0 +1,186 @@
+package deploy
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	apiextv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+// initDeployStructFields queries the users' shell environment to initialize
+// the deploy structure fields to customize the metering deploy process.
+// In the future, the logic of setting the deploy fields moves to cli flags/options
+// and less reliant on environment variables to customize a metering instance.
+func (deploy *Deploy) initDeployStructFields() error {
+	meteringNamespace := os.Getenv("METERING_NAMESPACE")
+	if meteringNamespace == "" {
+		return fmt.Errorf("error: you need to set the $METERING_NAMESPACE env var")
+	}
+
+	deployType := os.Getenv("DEPLOY_TYPE")
```

Comment:
I think whether we Install or Uninstall be handled in `main()` and we should remove `Type` as a field on `Deploy` entirely.